README.md

```diff
@@ -7,6 +7,7 @@ manifold manages, orchestrates, and displays data about Temple University Librar
 
 - Ruby 2.7.2
```

Comment:
Maybe we should update this version since we are making changes?

Reply:
@cdoyle-temple I think it would make sense to tell users to use whatever Ruby version is in the Gemfile so that you don't have to worry about updating the Readme whenever it changes.